packages/framework/src/Resources/views/Admin/Form/priceListProductsPickerType.html.twig

```diff
@@ -78,9 +82,9 @@
             <span class="js-price-list-product-picker-item-product-name">
                 {% if product is not null %}
                     <a
-                        href="{{ url('admin_product_edit', {id: product.id}) }}"
-                        target="_blank"
-                        class="btn-link-style btn-link-style--iconic-l"
+                            href="{{ url('admin_product_edit', {id: product.id}) }}"
+                            target="_blank"
+                            class="btn-link-style btn-link-style--iconic-l"
```

Comment:
nitpick, the formatting is weird, there are too many spaces...

Reply:
better? 89d8434f25